app/assets/javascripts/initializers/initializeDashboardSort.js

```diff
@@ -1,6 +1,8 @@
+'use strict';
+
 function initializeDashboardSort() {
   if (document.getElementById('dashhboard_sort')) {
-    document.getElementById('dashhboard_sort').onchange = function() {
+    document.getElementById('dashhboard_sort').onchange = () => {
```

Comment:
it'd be great if you could refactor this to use `addEventListener` instead of attaching the function directly

Reply:
sure will update it .

Reply:
@rhymes  i tries changing it to addEventListener but inside it also if i pass callback function as `function() {}` i.e without name it gives same eslint error unnamed function.

Reply:
or either I can give it a name or use the arrow function.

Reply:
please don't request a review before you changed the code :D I can still see when I'm mentioned. I think `addEventListener` plus arrow function is the way to go.

You'll need to replace `this` with the event target I guess

Reply:
ps. in the future maybe don't request a review before having updated the code :)